package.json

```diff
@@ -35,6 +35,7 @@
 	],
 	"devDependencies": {
 		"@sindresorhus/tsconfig": "~0.7.0",
+		"expect-type": "^0.9.0",
```

Comment:
Can you update to the latest version?

Reply:
Do we still want `expect-type` in there now that `tsd` is activated again as of #162?

Reply:
Yes, it covers some cases `tsd` currently does not handle.

Reply:
Will update the version and obviously no problem if this gets replaced with tsd in future.